robots/lwr_sdh.urdf.xacro

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="lwr">
+
+  <xacro:arg name="use_gazebo" default="1"/>
+  <xacro:arg name="use_transmission" default="1"/>
+  <xacro:arg name="collision_simple" default="1"/>
+  <xacro:arg name="visual_simple" default="0"/>
+
+  <!-- KUKA LWR4 arm -->
+  <xacro:include filename="$(find kuka_lwr4_description)/robots/kuka_lwr_arm.urdf.xml"/>
+
+  <!-- Schunk SDH2 hand -->
+  <xacro:include filename="$(find schunk_description)/urdf/sdh/sdh.urdf.xacro"/>
+  <xacro:include filename="$(find schunk_description)/urdf/materials.urdf.xacro"/>
+  <schunk_sdh name="sdh" parent="lwr_arm_7_link">
+    <origin rpy="0.0 0.0 1.57079" xyz="0 0 0.022"/>
```

Comment:
M_PI/2 - or if we say we go deprecate indigo and go jade and newer, we can use pi directly.